ApollosProject/apollos-apps
# Fixes Html content doubling issue

## DESCRIPTION

### What does this PR do, or why is it needed?

Content that updated was showing up twice. Most noticeable where there was data in the apollos cache that would render, followed by the actual data from the server.

Before: 
![2020-04-22 11 06 39](https://user-images.githubusercontent.com/1637694/80000765-8ed70c00-848b-11ea-84ad-8f4bd589de44.gif)


After
![2020-04-22 11 10 09](https://user-images.githubusercontent.com/1637694/80000758-8bdc1b80-848b-11ea-9aff-1dfd3e82d6d5.gif)

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

There's a story that simulates this issue in the HTMLView package. You should see the number in the story incrementing, rather than duplicating (see gifs above for examples)

### What automated tests did you write?

Tests over new functionality. 

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
